readyapi4j-core/src/test/java/com/smartbear/readyapi4j/TestRecipeBuilderTest.java

```diff
@@ -55,23 +58,20 @@ public void buildRecipeWithTestCaseProperties() {
 
     @Test
     public void buildsRecipeWithPropertyTransferTestStep() throws Exception {
-        TestRecipe recipe = newTestRecipe()
-                .addStep(propertyTransfer()
-                        .addTransfer(
-                                from(aSource()
-                                        .withSourceStep("sourceName")
-                                        .withProperty("username")
-                                        .withPath("sourcePath")
-                                        .withPathLanguage(PathLanguage.XPath)
-                                )
-                                        .to(aTarget()
-                                                .withTargetStep("targetName")
-                                                .withProperty("username")
-                                                .withPath("targetPath")
-                                                .withPathLanguage(PathLanguage.XPath)
-                                        )
+        TestRecipe recipe = newTestRecipe(propertyTransfer(from(aSource()
+                        .withSourceStep("sourceName")
```

Comment:
could we condense `aSource().withSourceStep("...")` to simply `source("...")` - with an optional shortcut `previous()` ?